src/components/swapv2/styleds.tsx

```diff
@@ -231,7 +231,7 @@ export function SwapCallbackError({ error }: { error: string }) {
           Show more details
         </Text>
         {showDetail && (
-          <Text color={theme.text1} fontSize="10px" style={{ margin: '10px 0 4px 0' }}>
+          <Text color={theme.text1} fontSize="10px" style={{ margin: '10px 0 4px 0', lineHeight: '16px' }}>
```

Comment:
Same as above, magin also is one of props of Text

Reply:
agree